.github/actions/run_unit_tests/action.yml

```diff
@@ -7,10 +7,6 @@ runs:
       run: poetry build
       shell: bash
 
-    - name: Install project whl
-      run: pip install dist/*.whl
```

Comment:
pip install needs now needs an additional package index definition, because we fetch the torch package not from pypi, but from torch's package index. Maybe, we should make the torch installation optional for the this package, such that the deployment scripts can be installed without needing to worry about torch.
Another, issue I see with install the whl that way, is that we don't control where we install it, too.